test/grpc/integration/interceptor_headers.exs

```diff
@@ -0,0 +1,60 @@
+defmodule GRPC.Integration.EndpointTest do
```

Comment:
Could you rename this file to `interceptor_headers_test.exs` and rename the module to `GRPC.Integration.InterceptorHeadersTest`? btw, it's better to rename them to "client_interceptor..."